File: Segundo Trimestre/5- Ejercicios Colecciones y Diccionarios/Java/Ejercicio3.java
/**
 * Escribe un programa que ordene 10 números enteros introducidos por teclado
 * y almacenados en un objeto de la clase ArrayList.
 * 
 * @author RafaLpeC
 * 
 */

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Scanner;

public class Ejercicio3 {
  public static void main(String[] args) {
  Scanner s = new Scanner (System.in);
  int i = 0;
  int n = 0;
  List<Integer> numero = new ArrayList<Integer>();
  while(i!=6) {
    try {
      System.out.println("Introduce un número: ");
      n = s.nextInt();
      numero.add(n);
      i++;
    }
    catch (Exception e) {
    System.out.println("¡Introduce un número correcto!");
    s.next();
    }
  }
  Collections.sort(numero);
  System.out.println("El número introducido más pequeño es " + numero.get(0));
  System.out.println("El número más grande es " + numero.get(5));
  }
}

File: Segundo Trimestre/5- Ejercicios Colecciones y Diccionarios/Python/Ejercicio3.py
"""
Ejercicio 3
Escribe un programa que ordene 10 números enteros introducidos por teclado y
 almacenados en un objeto de la clase ArrayList.
 
@author: RafaLpeC
"""

lista = []
i = 0
while(i!=6):
    try:
        lista.append(int(input("Introduzca un número a la lista: ")))
        i = i+1
    except Exception:
        print("¡Introduce un número correcto!")

lista.sort()
print("El número introducido más pequeño es "+str(lista[0]))
print("El número introducido más grande es "+str(lista[5]))